data/language/en-GB.txt

```diff
@@ -4114,6 +4114,7 @@ STR_5805    :{SMALLFONT}{BLACK}If checked, your server will be added to the{NEWL
 STR_5807    :{WINDOW_COLOUR_2}Number of rides: {BLACK}{COMMA16}
 STR_5808    :{WINDOW_COLOUR_2}Number of shops and stalls: {BLACK}{COMMA16}
 STR_5809    :{WINDOW_COLOUR_2}Number of information kiosks and other facilities: {BLACK}{COMMA16}
+STR_5810    :{SMALLFONT}{BLACK}Makes the ride brand new.{NEWLINE}Resets ride age and reliability
```

Comment:
Given you have implemented this like a gameplay feature, I find this far too technical for a user tooltip. It should say something like, refurbishes the ride.